src/components/manage/Blocks/Listing/ListingStyle.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { blocks } from '~/config';
+import { compose } from 'redux';
+import withObjectBrowser from '@plone/volto/components/manage/Sidebar/ObjectBrowser';
+import TemplateWidget from './TemplateWidget';
+
+const ListingStyle = ({ data, block, onChangeBlock, required = false }) => {
+  const templatesConfig = blocks?.blocksConfig?.listing?.templates;
+
+  if (templatesConfig && Object.keys(templatesConfig).length > 1) {
+    return (
+      <div className="sidebar-listing-data listing-style">
+        <TemplateWidget
+          data={data}
+          block={block}
+          onChangeBlock={onChangeBlock}
+          required={required}
+        />
+      </div>
+    );
+  }
+
+  return null;
+};
+
+ListingStyle.propTypes = {
+  data: PropTypes.objectOf(PropTypes.any).isRequired,
+  block: PropTypes.string.isRequired,
+  onChangeBlock: PropTypes.func.isRequired,
+  openObjectBrowser: PropTypes.func.isRequired,
+};
+
+export default compose(withObjectBrowser)(ListingStyle);
```

Comment:
Is the object browser possibly here by mistake? Or is it used somewhere I didn't notice?

Reply:
no it's a mistake 😢